src/dialect/mod.rs

```diff
@@ -349,6 +349,7 @@ mod tests {
             }
         }
 
+        #[allow(clippy::needless_raw_string_hashes)]
```

Comment:
maybe we can fix this in a follow on PR (as in remove the clippy allow and then fix the tests to conform to the new clippy opinions)

Reply:
Yeah, is a good idea. I tried to fix, but without the raw the parser errors with `Unterminated string literal`. 
I wasn't sure enough of the underlying intent to make more a dramatic change.